raiden/utils/testnet.py

```diff
@@ -0,0 +1,71 @@
+from typing import TYPE_CHECKING
+
+from eth_utils import to_checksum_address
+
+from raiden.exceptions import RaidenError, TransactionNotSuccessful
+from raiden.network.rpc.transactions import check_transaction_threw
+from raiden.utils.typing import Any, List, TokenAddress, TransactionHash
+
+if TYPE_CHECKING:
+    # pylint: disable=unused-import
+    from raiden.network.rpc.client import JSONRPCClient
+    from raiden.network.rpc.smartcontract_proxy import ContractProxy
+
+
+_MINT_ABI = [
+    dict(
+        name="increaseSupply",
+        type="function",
+        constant=False,
+        payable=False,
+        stateMutability="nonPayable",
+        inputs=[dict(name="value", type="uint256"), dict(name="to", type="address")],
+        outputs=[],
+    ),
+    dict(
+        name="mint",
+        type="function",
+        constant=False,
+        payable=False,
+        stateMutability="nonPayable",
+        inputs=[dict(name="to", type="address"), dict(name="value", type="uint256")],
+        outputs=[],
+    ),
+    dict(
+        name="mintFor",
+        type="function",
+        constant=False,
+        payable=False,
+        stateMutability="nonPayable",
+        inputs=[dict(name="num", type="uint256"), dict(name="target", type="address")],
+        outputs=[],
+    ),
+]
+
+
+def token_minting_proxy(client: "JSONRPCClient", address: TokenAddress) -> "ContractProxy":
+    return client.new_contract_proxy(
+        contract_interface=_MINT_ABI, contract_address=to_checksum_address(address)
+    )
+
+
+def call_contract_method(
+    client: "JSONRPCClient", proxy: "ContractProxy", method: str, args: List[Any]
+) -> TransactionHash:
+    gas_limit = proxy.estimate_gas("latest", method, *args)
+    if gas_limit is None:
+        raise TransactionNotSuccessful(
+            f"Gas estimation failed. Make sure the token has a method "
+            f"named {method} with the expected signature."
+        )
+
+    try:
+        tx_hash = proxy.transact(method, gas_limit, *args)
+    except (RaidenError, ValueError) as e:
```

Comment:
In which cases are `RaidenError` and `ValueError` going to be raised? Can you add a docstring here to clarify this?

Reply:
I added a comment